server/model/adjustmentsHubViewModel.ts

```diff
@@ -46,20 +45,20 @@ export default class AdjustmentsHubViewModel {
     return adjustmentTypes.filter(it => !it.deduction)
   }
 
-  public hasRemandToolRole(): boolean {
-    return this.roles.indexOf('REMAND_IDENTIFIER') !== -1
-  }
-
   public showProspectiveAdaLink(adjustmentType: AdjustmentType): boolean {
     return adjustmentType.value === 'ADDITIONAL_DAYS_AWARDED' && !!this.adaAdjudicationDetails?.prospective?.length
   }
 
   public displayAddLink(adjustmentType: AdjustmentType): boolean {
-    return (
-      !this.hasRemandToolRole() ||
-      adjustmentType.value !== 'REMAND' ||
-      (this.remandDecision && !this.remandDecision.accepted)
-    )
+    return !this.hasRemandToolRole() || adjustmentType.value !== 'REMAND' || this.isRemandDecisionNotAccepted()
```

Comment:
not sure what reads better
```suggestion
    return !this.hasRemandToolRole() || !this.isRemand(adjustmentType) || this.isRemandDecisionNotAccepted()
```